.circleci/images/integration/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 the u-root Authors. All rights reserved
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# Build:
+#
+#     sudo docker build . -t uroot-integration
+#
+# Run:
+#
+#     sudo docker run --rm -it uroot-integration
+#
+# More instructions:
+#
+#     https://circleci.com/docs/2.0/custom-images/
+
+FROM circleci/golang:1.10.3
+
+# Install dependencies
+RUN sudo apt-get update &&                          \
+    sudo apt-get install -y --no-install-recommends \
+        `# Linux dependencies`                      \
+        git                                         \
+        bc                                          \
+        bison                                       \
+        flex                                        \
+        gcc                                         \
+        make                                        \
+        `# QEMU dependencies`                       \
+        libglib2.0-dev                              \
+        libfdt-dev                                  \
+        libpixman-1-dev                             \
+        zlib1g-dev &&                               \
+    sudo rm -rf /var/lib/apt/lists/*
+
+# Create working directory
+WORKDIR /home/circleci
+COPY config_linux4.17_amd64.txt .config
+
+# Build linux
```

Comment:
Sorry I'm confused, are you rebuilding linux every time then?

Reply:
I built the Linux kernel once last night, then uploaded it to Docker Hub. CircleCI downloads it from Docker Hub (and likely caches it).

See: https://circleci.com/docs/2.0/custom-images/